### PR TITLE
✨ Introduce isChicken() helper

### PR DESCRIPTION
**Pull request by euzuro**
# Context:

# To Reproduce:
- step1
- step2

### CODE REVIEW

File: foo/bar.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// This is a sample file for screenshot purposes
+
+
+export function isChicken(term: string): boolean {
+    if (term == "chicken") {
+	return true;
+    }
```
Comment (euzuro) [inline foo/bar.js:7]: chore:

Can we switch this to use triple equals `===` ?

```suggestion
    if (term === "chicken") {
	return true;
    }
```

File: foo/bar.js [hunk 2]
```diff
@@ -13,3 +13,12 @@ export function isChicken(term: string): boolean {
     }
     return false;
 }
+
+
+export function getValue(term: string): number {
+    if (isChicken(term)) {
+	return 2000;
```
Comment (euzuro) [inline foo/bar.js:20]: chore:

Can we make that a constant?
Maybe `const CHICKEN_VALUE = 2000;` ?

File: foo/bar.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    return false;
+}
+
+
+export function getValue(term: string): number {
+    if (isChicken(term)) {
+	return 2000;
+    } else {
+	return 0;
+    }
```
Comment (euzuro) [inline foo/bar.js:23]: suggestion:

We could use a ternary here to be more concise:

```suggestion
    return isChicken(term) ? 2000 : 0;
```

File: foo/bar.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    return false;
+}
+
+
+export function getValue(term: string): number {
+    if (isChicken(term)) {
+	return 2000;
+    } else {
+	return 0;
```
Comment (euzuro) [inline foo/bar.js:22]: question:

I know chickens are the best, but is everything else really _0_?
Do we have some docs/specs I could read about this?

File: foo/bar.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+// This is a sample file for screenshot purposes
+
+
+export function isChicken(term: string): boolean {
+    switch (term) {
```
Comment (euzuro) [inline foo/bar.js:5]: praise:

Nice use of `switch()`! Makes this easy to extend to other languages.

File: foo/bar.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        case "chicken":
```
Comment (euzuro) [inline foo/bar.js:6]: Are you sure this is a good idea?

Comment (euzuro) [reply foo/bar.js:6]: Yeah I'm pretty sure.

Comment (euzuro) [inline foo/bar.js:6]: Are you sure we want "chicken" here?

Comment (anediaz) [reply foo/bar.js:6]: Yes, this is what I want here

Comment (euzuro) [reply foo/bar.js:6]: Yeah but "chicken" might not always be the right way to say this.

Comment (anediaz) [reply foo/bar.js:6]: If you look at the specs that's what it says!

Comment (euzuro) [reply foo/bar.js:6]: Which specs are you reading? Can you send a link?

**Pull request closed without merging**